CHANGELOG.md

```diff
@@ -3,6 +3,85 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.16.0
+_Sep 28, 2023_
+
+We'd like to offer a big thanks to the 9 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Added `clearable` prop to DatePickers (#9095) @noraleonte
+- 🚀 Fix header filters menu auto closing on render (#10483) @MBilalShafi
+- 🎯 Fix column headers scroll when theme scoping is used (#10437) @cherniavskii
+- 💫 Add customization playground (#9581) @noraleonte
+
+You can play around with style customization options on the [Date Picker documentation](https://mui.com/x/react-date-pickers/date-picker/#customization).
+
+We are thrilled to hear your feedback about this functionality!
+
```

Comment:
Could we move this to the top or after the first detailed highlight?
We try to put the more emphasized highlights with detailed explanation at the top.